docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +262,30 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+1. To view metrics in Dynatrace dashboards, import the file [Telemetry Module Status](./telemetry-resource-metrics.json) as a Dynatrace dashboard. For details, see [Importing Dashboards](https://docs.dynatrace.com/docs/analyze-explore-automate/dashboards-classic/dashboards/dashboard-json#import-dashboard).
+
+2. Add the following attributes to the allow list of OpenTelemetry metrics resource attributes:
```

Comment:
The metrics for the Telemetry module health are using custom resource attributes, which needs to be added to ...

Reply:
Why passive voice? How about:
```suggestion
2. Add the following custom resource attributes to the allow list of OpenTelemetry metrics resource attributes:
```
I'm not sure why we need the first half of the sentence. What value does it add?

Reply:
I like speaking in passive voice, no other intention here.

Reply:
"Speaking in passive voice is liked by me!" ;)

Speaking in passive voice is absolutely fine ...but let's use active voice in docs.